app/api/threads/update_thread.robustness.feature

```diff
@@ -74,35 +74,35 @@ Feature: Update thread - robustness
       | item_id | participant_id | status | helper_group_id | latest_update_at |
 
   Scenario: Should be logged
-    When I send a PUT request to "/items/10/participant/1/thread"
+    When I send a PUT request to "/items/@SomeItem/participant/@RichardFeynman/thread"
```

Comment:
I think I understand the initial idea but in practice, if I read that, I don't know what ids are used for the requests and so I cannot really check if the behavior is correct. There is no even RichardFeynman in the db on top, what is given as value used in practice in this test for `@SomeItem`, are we sure the rule was not interpret as the string "@SomeItem" here which would explain the 404 ?...  it makes things a bit obscure and so do not really help.
That's just an opinion, we can discuss this.